src/sources/video_card.test.ts

```diff
@@ -0,0 +1,36 @@
+import getVideoCard from './video_card'
+
+describe('Sources', () => {
+  describe('videoCard', () => {
+    const videoCard = getVideoCard()
```

Comment:
The code to test should be inside the `it` statements

Reply:
This notice is actual again.

Also please use `isGecko` from `'../../tests/utils'`.

Reply:
Done !